spec/jobs/decidim/action_delegator/admin/invite_participants_job_spec.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+RSpec.describe Decidim::ActionDelegator::Admin::InviteParticipantsJob, type: :job do
+  let(:organization) { create(:organization) }
+  let(:current_setting) { create(:setting, consultation: consultation) }
+  let(:consultation) { create(:consultation) }
+  let!(:participants) { create_list(:participant, 3, setting: current_setting) }
+
+  it "invites participants" do
+    participants.each do |participant|
+      expect { described_class.perform_later(participant, organization) }.not_to raise_error
+    end
+  end
```

Comment:
Can we add a test to check that the email si being sent?
something like this (you'll find more examples in the Decidim codebase):

```ruby
      it "sends an email to the invited participant" do
        described_class.perform_now(...)

        email = last_email
        expect(email.subject).to include("...)
         ...
      end

    context "when participants are registered" do
      ..
      it "does not send invitiations" do
        ...
```